text-summarization/custom-recipes/text-summarization-compute/recipe.py

```diff
@@ -6,22 +6,28 @@
 import dataiku
 from dataiku.customrecipe import *
 import numpy as np
+import os
+import sys
 
 from sumy.parsers.plaintext import PlaintextParser
 from sumy.nlp.tokenizers import Tokenizer
 from sumy.nlp.stemmers import Stemmer
 from sumy.utils import get_stop_words
 
 import nltk
-nltk.download('punkt')
+#nltk.download('punkt')
```

Comment:
I don't see the need to keep commented code in the code base with version control. Obviously, people have different ways of dealing with this, but especially with the inline import below it seems unlikely it will need to be added again